packages/dashboard/src/components/DashboardGridLayout/DashboardGridLayoutWithAddCardPanel/DashboardGridLayoutWithAddCardPanel.types.ts

```diff
@@ -50,6 +50,8 @@ export interface IDashboardGridLayoutWithAddCardPanelProps extends IBaseProps {
    * ref passed. If ref is not passed scrolling is applied to the immediate parent element
    */
   scrollElement?: HTMLElement;
+
+  emptyCardContent?: string;
```

Comment:
All the props related to add card panel are in IAddCardPanelProps interface below.It's better if we move this to IAddCardPanelProps. That way all props related to AddCardPanel stick together.Also emptyCardContent does not look like a proper name for the prop. Could you change that to emptyPanelMessage and please add comments for this as well